src/componentprocessor/index.ts

```diff
@@ -156,24 +156,6 @@ async function processXmlTree(
         if (xmlNode) {
             nodeDef.children = getChildren(xmlNode);
             nodeDef.scripts = getScripts(xmlNode);
-            let baseNode = xmlNode.attr.extends;
-            while (baseNode) {
-                let baseNodeDef = nodeDefMap.get(baseNode);
-                if (baseNodeDef) {
-                    nodeDef.children = [...nodeDef.children];
```

Comment:
It turns out this whole `while` loop doesn't actually do anything, because
```
nodeDef.children = [...nodeDef.children]
```
is a noop!